src/webui/www/public/scripts/mocha-init.js

```diff
@@ -451,6 +451,36 @@ initializeWindows = function() {
         }
     };
 
+    copyNameFN = function() {
+        var selectedRows = torrentsTable.selectedRowsIds();
+        var names = [];
+        if (selectedRows.length) {
+            var rows = torrentsTable.getFilteredAndSortedRows();
+            for (var i = 0; i < selectedRows.length; i++) {
+                var hash = selectedRows[i];
+                names.push(rows[hash].full_data.name);
+            }
+        }
+        return names.join("\n");
+    };
+
+    copyMagnetLinkFN = function() {
+        var h = torrentsTable.selectedRowsIds();
```

Comment:
I'll do the [fixups](https://github.com/qbittorrent/qBittorrent/pull/7036#discussion_r128156049) myself.